pkg/infradriver/ssh_infradriver.go

```diff
@@ -306,7 +306,7 @@ func (d *SSHInfraDriver) GetClusterLaunchApps() []string {
 }
 
 func (d *SSHInfraDriver) GetHostName(hostIP net.IP) (string, error) {
-	hostName, err := d.CmdToString(hostIP, nil, "hostname", "")
+	hostName, err := d.CmdToString(hostIP, nil, "uname", "-n")
```

Comment:
d.CmdToString(hostIP, nil, "uname -n", "")

Reply:
My bad. I've fixed it.